CONTRIBUTORS.md

```diff
@@ -55,7 +55,7 @@ Cerner Corporation
 [@pseudofaux]: https://github.com/pseudofaux
 [@bjankord]: https://github.com/bjankord
 [@eawww]: https://github.com/eawww
-[@mndrsn]: https://github.com/mndrsn
+[@mndrsn]: https://github.com/mndrsnq
```

Comment:
whoops, looks like a typo go in here

Reply:
I fixed the typo. https://github.com/cerner/terra-clinical/pull/479/commits/ae0e79b4d988bea9f0678a7b9285cb03e6b279fe